src/data.ts

```diff
@@ -0,0 +1,71 @@
+export interface BookInfo {
+  key: string;
+  title: string;
+  cover_i?: number;
```

Comment:
Can you explain, why `cover_i`? Can you give more descriptive name?

Reply:
This naming comes from backend. On week 2 I divided it for api response models and data models which I used in my code.